### ISSUE TITLE
Not able to Register user via Gmail or GitHub or simply user data

### ISSUE THREAD

**Issue by Sh00bham** [rajeshmn47/dream-11-clone#50]
Not able to register the user even though I have connected Mongodb.

Also can you please share the workflow of the project

**Comment by rajeshmn47** [comment]
their is backend,dreamelevenclonebackend search it,run it

**Comment by rajeshmn47** [comment]
set up .env

**Comment by vmehta0387** [comment]
Hi Rajesh, can you share me dreamelevenclonebackend  please?

**Comment by rajeshmn47** [comment]
Please give support to run backend. I will give it but can u support the project?

**Comment by rajeshmn47** [comment]
Yes thank you. U can whatsapp me. Give your number I will do it.

**Comment by vmehta0387** [comment]
Provided you my number on your email.
